Sage-Bionetworks/Bridge-Exporter
# V2 instant export

all unit tests done;
manual tests done;
integ tests done;

See Also:
* https://github.com/Sage-Bionetworks/BridgeJavaSDK/pull/393
* https://github.com/Sage-Bionetworks/BridgeExporterIntegrationTests/pull/1
* https://github.com/Sage-Bionetworks/BridgePF/pull/1393
* https://github.com/Sage-Bionetworks/BridgeDocs/pull/138
* https://github.com/Sage-Bionetworks/Bridge-EX-Scheduler/pull/6
